### ISSUE TITLE
Chapter 6 - Phong.frag, missing clamp call

### ISSUE THREAD

**Issue by lfridael** [gameprogcpp/code#36]
When discussing the `Phong.frag`shader the book mentions using `clamp` to limit the Phong RGB components to the 0.0-1.0 range. 

The actual `clamp` call seems to be missing from the shader code, however:
https://github.com/gameprogcpp/code/blob/master/Chapter06/Shaders/Phong.frag

Adding this, the visual result is markedly different:

```glsl
 Phong = clamp(Phong, 0.0, 1.0); 
```

**Comment by chalonverse** [comment]
Good catch, I actually don't remember whether that's intentional or if it was a mistake!
